### PR TITLE
Add MobaLiveCD.txt to RApps database

### PR DESCRIPTION
**Pull request by R0shyx**
It is an emulator tool so I put category as "12".

virtualbox_reactosmobalivecd6
virtualbox_reactosmobalivecd7
virtualbox_reactosmobalivecd11

### CODE REVIEW

**Comment by R0shyx** [comment]
Despite some glitches it works.

File: MobaLiveCD.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[Section]
+ Name = MobaLiveCD
```
Comment (binarymaster) [inline MobaLiveCD.txt]: Whitespace in beginning of the string.

File: MobaLiveCD.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
+[Section]
+ Name = MobaLiveCD 
+Version = 2.1
+License = GPL
+
+ Description = MobaLiveCD is a freeware that will run your Linux LiveCD on Windows thanks to the excellent emulator called "Qemu".
```
Comment (binarymaster) [inline MobaLiveCD.txt]: Ditto.

File: MobaLiveCD.txt [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ 
+Size = 1.52MB
+ 
+Category = 12
+URLSite = https://www.mobatek.net/labs_mobalivecd.html
+URLDownload = https://www.mobatek.net/exes/MobaLiveCD_v2.1.exe
+CDPath = none
+ 
+
```
Comment (binarymaster) [inline MobaLiveCD.txt]: Some suspicious empty lines.

File: MobaLiveCD.txt [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+Size = 1.52MB
+ 
+Category = 12
+URLSite = https://www.mobatek.net/labs_mobalivecd.html
+URLDownload = https://www.mobatek.net/exes/MobaLiveCD_v2.1.exe
+CDPath = none
+ 
+ 
+
+Size Bytes= 1.601.024 bytes
```
Comment (binarymaster) [inline MobaLiveCD.txt]: Typo in parameter name.

File: MobaLiveCD.txt [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+[Section.0415]
+ Description = MobaLiveCD is a freeware that will run your Linux LiveCD on Windows thanks to the excellent emulator called "Qemu". MobaLiveCD allows you to test your LiveCD with a single click.
```
Comment (binarymaster) [inline MobaLiveCD.txt]: Whitespace.

"Qemu" should be uppercase as below.

File: MobaLiveCD.txt [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+[Section.040a]
+ Description = MobaLiveCD es un programa freeware que correrá tus LiveCD de Linux en Windows debido al excelente emulador llamado "QEMU". MobaLiveCD te permite  probar tus LiveCD con un simple click.
```
Comment (binarymaster) [inline MobaLiveCD.txt]: Ditto.

File: MobaLiveCD.txt (hunk 3 above)
Comment (R0shyx) [reply MobaLiveCD.txt]: Sorry for the problems. Feel free to correct the wrong things you find.

File: MobaLiveCD.txt [hunk 7]
```diff
@@ -0,0 +1,16 @@
+[Section]
+Name = MobaLiveCD 
+Version = 2.1
+License = GPL
+Description = MobaLiveCD is a freeware that will run your Linux LiveCD on Windows thanks to the excellent emulator called "Qemu".
```
Comment (HBelusca) [inline MobaLiveCD.txt]: `thanks to the excellent "Qemu" emulator.`

File: MobaLiveCD.txt [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+Size = 1.52MB
+Category = 12
+URLSite = https://www.mobatek.net/labs_mobalivecd.html
+URLDownload = https://www.mobatek.net/exes/MobaLiveCD_v2.1.exe
+CDPath = none
+Size Bytes= 1.601.024 bytes
+
+[Section.0415]
+ Description = MobaLiveCD is a freeware that will run your Linux LiveCD on Windows thanks to the excellent emulator called "Qemu". MobaLiveCD allows you to test your LiveCD with a single click.
```
Comment (HBelusca) [inline MobaLiveCD.txt]: `thanks to the excellent "Qemu" emulator.`

File: MobaLiveCD.txt [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+[Section.040a]
+ Description = MobaLiveCD es un programa freeware que correrá tus LiveCD de Linux en Windows debido al excelente emulador llamado "QEMU". MobaLiveCD te permite  probar tus LiveCD con un simple click.
```
Comment (HBelusca) [inline MobaLiveCD.txt]: Remove the whitespace before `Description =`

File: MobaLiveCD.txt (hunk 8 above)
Comment (HBelusca) [inline MobaLiveCD.txt]: Remove the whitespace before `Description =`

File: MobaLiveCD.txt (hunk 4 above)
Comment (HBelusca) [reply MobaLiveCD.txt]: And add a space between the parameter name and the "equals" sign.

**Comment by R0shyx** [comment]
I correct some white spaces, so it should be fine now. I was hurry....
Feel free to correct/fix what do you consider is wrong... maybe the description. The other things like website and size is correct.
Regards.

File: MobaLiveCD.txt (hunk 7 above)
Comment (ThFabba) [reply MobaLiveCD.txt]: Both seem equally fine...

**Comment by ThFabba** [comment]
We generally require that you test the changes you propose -- can you please provide a screenshot of rapps showing this new entry? Thanks.

**Comment by R0shyx** [comment]
> We generally require that you test the changes you propose -- can you please provide a screenshot of rapps showing this new entry? Thanks.

Here´s the screenshot: 

mobalivecdtxt

**Comment by R0shyx** [comment]
I don´t know why so many problems. Just correct the text if I am offline.

**Comment by HBelusca** [comment]
>> We generally require that you test the changes you propose -- can you please provide a screenshot of rapps showing this new entry? Thanks.
>
>Here´s the screenshot:

He asked you to show RAPPS (running in ReactOS) showing the entry in question.

**Comment by R0shyx** [comment]
> > > We generally require that you test the changes you propose -- can you please provide a screenshot of rapps showing this new entry? Thanks.
> > 
> > 
> > Here´s the screenshot:
> 
> He asked you to show RAPPS (running in ReactOS) showing the entry in question.

I submitted the screenshots of MobaLiveCD running in ReactOS.  It is a portable emulator that uses QEMU.

**Comment by ThFabba** [comment]
What I asked for was a screenshot showing your MobaLiveCD.txt file loaded in the ReactOS Application Manager.
The reason for this is simple: you are proposing a change to this repository, and I'm asking for evidence that you've tested this change. We will do our best to review and find problems but if you don't run a sanity check against patches you're proposing, it is wasteful of our time.

**Comment by R0shyx** [comment]
> What I asked for was a screenshot showing your MobaLiveCD.txt file loaded in the ReactOS Application Manager.
> The reason for this is simple: you are proposing a change to this repository, and I'm asking for evidence that you've tested this change. We will do our best to review and find problems but if you don't run a sanity check against patches you're proposing, it is wasteful of our time.

Yes. I didn´t understand before. Sorry.
I can´t find MobaLiveCD text loaded in RApps, there is no evidence of it.

**Comment by R0shyx** [comment]
I edited the description.
Will this pull request be merged or not?

**Comment by learn-more** [comment]
Yes, you were asked to show the entry in rapps here, and that also goes for your other pr.

**Comment by R0shyx** [comment]
> Yes, you were asked to show the entry in rapps here, and that also goes for your other pr.

I saw RApps on an installed ReactOS under VirtualBox and there is no "MobaLiveCD" entry in database applications.  
And here is from my fork https://github.com/R0shyx/rapps-db/blob/master/MobaLiveCD.txt
If it is no what you are saying, I have no idea what are you taking about.

Adding ZinjaI was so easy, why this PR is so difficult?

**Comment by ThFabba** [comment]
You need to copy the txt file to `C:\Documents and Settings\Administrator\Local Settings\rapps\rapps` in ROS or the appropriate equivalent path in Windows.

**Comment by binarymaster** [comment]
> Adding ZinjaI was so easy , and this PR is so difficult...

It was accidentally merged without proper review.

**Comment by R0shyx** [comment]
> > Adding ZinjaI was so easy , and this PR is so difficult...
> 
> It was accidentally merged without proper review.

But it is done finally. 
Or will you delete ZinjaI entry?

**Comment by learn-more** [comment]
Hello,
Thanks for your interest in ReactOS!
We ask our contributors [to use their full name when committing](https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations).
[Please amend your commit with your full name](https://stackoverflow.com/a/3042512), and update this PR.

Regards,

**Comment by C4rl0b** [comment]
> Hello,
> Thanks for your interest in ReactOS!
> We ask our contributors [to use their full name when committing](https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations).
> [Please amend your commit with your full name](https://stackoverflow.com/a/3042512), and update this PR.
> 
> Regards,

I updated my profile and I am on it with PR...

**Comment by R0shyx** [comment]
> > Hello,
> > Thanks for your interest in ReactOS!
> > We ask our contributors [to use their full name when committing](https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations).
> > [Please amend your commit with your full name](https://stackoverflow.com/a/3042512), and update this PR.
> > Regards,
> 
> I updated my profile and I am on it with PR...

So anonymous code and + contributions is striclty prohited. 
Do you have a way to check even if a name, and account is real?

**Comment by R0shyx** [comment]
Well one day will be reopened or someone else will do it. 
hbeluscacomment

**Comment by R0shyx** [comment]
Here´s the probe link: https://chat.reactos.org/reactos/pl/gsg735byetr45dfmi574pw6aac

**Pull request closed without merging**